releases.yml

```diff
@@ -1,4 +1,39 @@
 releases:
+  ec.3:
+    assembly:
+      basis:
+        brew_event: 52321997
+        reference_releases:
+          aarch64: 4.14.0-0.nightly-arm64-2023-07-05-191024
+          ppc64le: 4.14.0-0.nightly-ppc64le-2023-07-05-191024
+          s390x: 4.14.0-0.nightly-s390x-2023-07-05-191023
+          x86_64: 4.14.0-0.nightly-2023-07-05-191022
+      group:
+        upgrades: 4.13.0-rc.3,4.13.0-rc.4,4.13.0-rc.5,4.13.0-rc.6,4.13.0-rc.7,4.13.0-rc.8,4.13.0,4.13.1,4.13.2,4.13.3,4.13.4,4.14.0-ec.0,4.14.0-ec.1,4.14.0-ec.2
```

Comment:
let's also include 4.13.5 in case it gets promoted

Reply:
Ack

Reply:
Done